Library/Homebrew/list.sh

```diff
@@ -4,32 +4,38 @@
 homebrew-list() {
   case "$1" in
     # check we actually have list and not e.g. listsomething
-    list) ;;
-    list*) return 1 ;;
+    list | ls) ;;
+    list* | ls*) return 1 ;;
     *) ;;
   esac
 
   local ls_args=()
   local formula=""
   local cask=""
 
-  local first_arg
-  for arg in "$@"
+  local OPTIND=2 # skip $1 (and localise OPTIND to this function)
+  while getopts ":1lrt-:" arg
```

Comment:
Can we use this in the other similar commands/files like this rather than just for `ls`?

Reply:
Maybe, though I'm unlikely to be able to prioritise doing so this week.

Reply:
Yup, that's fine, could you open a `help wanted` issue just summarising this instead?

Reply:
Sure